Src/BootCamp.Chapter/UserInteraction.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public static class UserInteraction
+    {
+        public static void DisplayMenu()
+        {
+            ConsoleInit();
+            DisplayHeader();
+
+            Console.WriteLine("(1) Encrypt message");
+            Console.WriteLine("(2) Decrypt message");
+            Console.WriteLine("(3) Crack encrypted message");
+            Console.WriteLine();
+            Console.WriteLine("(9) Settings");
+            Console.WriteLine("(0) Exit program!");
+
+            ChooseMenu();
+        }
+
+        private static void ChooseMenu()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The Cyclomatic Complexity of this method is 12 which is greater than 10 authorized.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5012201)